src/praisonai-agents/praisonaiagents/mcp/mcp_http_streaming.py

```
@@ -0,0 +1,217 @@
+"""
+HTTP-Streaming client implementation for MCP (Model Context Protocol).
+Provides HTTP chunked streaming transport as an alternative to SSE.
+"""
+
+import asyncio
+import logging
+import threading
+import queue
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)